### ISSUE TITLE
No Projects Found vs. loading....

### ISSUE THREAD

**Issue by jSylvestre** [ucdavis/Harvest#625]
It seems odd to me to see No Projects found message, then the projects load.

**Comment by srkirkland** [comment]
Makes sense -- i think this would be easy to fix by making the state variable `Project[] | undefined` and defaulting to `undefined`.  Then we could check and if undefined, show loading, if 0, show to projects found, otherwise show projects.  I'll assign to @mxthu313.
